### PR TITLE
add machine learning section to Teams

### PR DESCRIPTION
**Pull request by WenJoanne**
add machine learning section to Teams

### CODE REVIEW

File: Teams/README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ specifically improving it and guiding it to become a 3rd platform for users to e
 experience. We're are also responsible for ensuring Fishbrain is the first and final resource people
 find when searching for fishing on the web.
 
-## Team ML
+## Team Machine Learning
+The Machine Learning team on one hand is responsible for utilizing our accumulated data advantage with AI technology to better serve our users. This means more advanced ways to supply services to help fulfil our customers fishing dreams in terms of efficiency, economy, and enjoyment. And on the other hand, Machine learning team is responsible for supporting the company using AI technology to help the company be more efficient and make more informed decisions on the business side.
```
Comment (lhansford) [inline Teams/README.md:33]:
```suggestion
The Machine Learning team on one hand is responsible for utilizing our accumulated data advantage with AI technology to better serve our users. This means more advanced ways to supply services to help fulfil our customers fishing dreams in terms of efficiency, economy, and enjoyment. And on the other hand, the Machine Learning team is responsible for supporting the company using AI technology to help the company be more efficient and make more informed decisions on the business side.
```

**Review by lhansford — APPROVED** [review]
👌

File: Teams/README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ specifically improving it and guiding it to become a 3rd platform for users to e
...
+It includes two sides of tasks: building machine learning applications together with product team; Analyzing and modelling data to support finance, product, marketing and other functions to make smarter and better decisions.
```
Comment (bundeeteddee) [inline Teams/README.md:34]:
```suggestion
It includes two types of tasks: building machine learning applications together with the product teams; Analysing and modelling data to support finance, product, marketing and other functions to make better and smarter decisions.
```

File: Teams/README.md [hunk 3]
```diff
@@ -29,6 +29,8 @@ specifically improving it and guiding it to become a 3rd platform for users to e
 experience. We're are also responsible for ensuring Fishbrain is the first and final resource people
 find when searching for fishing on the web.
 
-## Team ML
+## Team Machine Learning
+The Machine Learning team on one hand is responsible for utilizing our accumulated data advantage with AI technology to better serve our users. This means more advanced ways to supply services to help fulfil our customers fishing dreams in terms of efficiency, economy, and enjoyment. And on the other hand, the Machine Learning team is responsible for supporting the company using AI technology to help the company be more efficient and make more informed decisions on the business side.
+It includes two types of tasks: building machine learning applications together with the product teams; Analysing and modelling data to support finance, product, marketing and other functions to make better and smarter decisions.
```
Comment (mattias-lundell) [inline Teams/README.md:34]: `modelling` vs `modeling`? us is one `l`?

**Review by mattias-lundell — APPROVED** [review]

File: Teams/README.md (hunk 3 above)
Comment (WenJoanne) [reply Teams/README.md:34]: It was changed into two "ll" by Mac notes auto correction :) 
Yeah, I think it's American English that uses one l.

**Pull request merged**